### ISSUE TITLE
regression on 3.16.0

### ISSUE THREAD

**Issue by sibelius** [elastic/apm-agent-nodejs#2110]
**Describe the bug**

```jsx
node_modules/elastic-apm-node/lib/logging.js:142
...ecsFormat({ apmIntegration: false })
^

TypeError: ecsFormat is not a function
at Object.createLogger (app/node_modules/elastic-apm-node/lib/logging.js:142:8)
at new Config (app/node_modules/elastic-apm-node/lib/config.js:260:27)
at config (app/node_modules/elastic-apm-node/lib/config.js:222:10)
at Agent._config (app/node_modules/elastic-apm-node/lib/agent.js:145:16)
at new Agent (app/node_modules/elastic-apm-node/lib/agent.js:38:8)
at Object.<anonymous> (app/node_modules/elastic-apm-node/index.js:5:18)
```

**To Reproduce**

bumped from 3.15.0 to 3.16.0

**Expected behavior**

it should not break

**Environment (please complete the following information)**

- OS: [e.g. Linux] linux docker, mac os x
- Node.js version: v14.16.1
- APM Server version: 
- Agent version:

**How are you starting the agent? (please tick one of the boxes)**

- [ ] Calling `agent.start()` directly (e.g. `require('elastic-apm-node').start(...)`)
- [x] Requiring `elastic-apm-node/start` from within the source code
- [ ] Starting node with `-r elastic-apm-node/start`

**Additional context**

- Agent config options 
  <details>
    <summary>Click to expand</summary>

    ```
    replace this line with your agent config options
    ```
  </details>
- `package.json` dependencies:
  <details>
    <summary>Click to expand</summary>

    ```
    replace this line with your dependencies section from package.json
    ```
  </details>

**Comment by trentm** [comment]
@sibelius Thanks for the report. I can't yet repro. Are you able to show me the output of this command in your ".../app" dir please?

```
npm ls @elastic/ecs-pino-format
```

**Comment by trentm** [comment]
In my repro attempt I have this:

```
% npm ls @elastic/ecs-pino-format
p@1.0.0 /Users/trentm/tmp/p
└─┬ elastic-apm-node@3.16.0
  └── @elastic/ecs-pino-format@1.1.2
```

**Comment by trentm** [comment]
More details on an attempted repro (I can't repro yet):

```
% docker run -ti --rm node:14 /bin/bash
...
root@072aa6477f80:/# mkdir app
root@072aa6477f80:/# cd app
root@072aa6477f80:/app# npm init -y
...
root@072aa6477f80:/app# npm install elastic-apm-node
...
root@072aa6477f80:/app# vi foo.js
root@072aa6477f80:/app# cat foo.js
var apm = require('elastic-apm-node').start()
setInterval(function hi() {
	const tx = apm.startTransaction('hi')
	console.log('hi')
	tx.end()
}, 1000);
root@072aa6477f80:/app# node foo.js
{"log.level":"error","@timestamp":"2021-06-15T15:01:44.443Z","log":{"logger":"elastic-apm-node"},"ecs":{"version":"1.6.0"},"message":"APM Server transport error (ECONNREFUSED): connect ECONNREFUSED 127.0.0.1:8200"}
{"log.level":"error","@timestamp":"2021-06-15T15:01:44.553Z","log":{"logger":"elastic-apm-node"},"ecs":{"version":"1.6.0"},"message":"APM Server transport error (ECONNREFUSED): connect ECONNREFUSED 127.0.0.1:8200"}
hi
{"log.level":"error","@timestamp":"2021-06-15T15:01:45.460Z","log":{"logger":"elastic-apm-node"},"ecs":{"version":"1.6.0"},"message":"APM Server transport error (ECONNREFUSED): connect ECONNREFUSED 127.0.0.1:8200"}
hi
{"log.level":"error","@timestamp":"2021-06-15T15:01:46.520Z","log":{"logger":"elastic-apm-node"},"ecs":{"version":"1.6.0"},"message":"APM Server transport error (ECONNREFUSED): connect ECONNREFUSED 127.0.0.1:8200"}
^C
```

I also tried backing up to `@elastic/ecs-pino-format@1.1.0`.

**Comment by sibelius** [comment]
I think the problem was having 2 versions of elastic apm node in our monorepo, tks

**Comment by trentm** [comment]
Okay, thanks. Let me know if you hit other problems. There *is* some potentially confusing interplay between `elastic-apm-node` and `@elastic/ecs-pino-format`. They *both attempt to use the other*.

https://github.com/elastic/ecs-logging-nodejs/issues/79 was perhaps the same issue (which was fixed in `@elastic/ecs-pino-format@1.1.2`).
